mautic/mautic
# Video removed from email editor buttons

| Q  | A
| --- | ---
| Bug fix? | Y
| New feature? | N
| Related user documentation PR URL | /
| Related developer documentation PR URL | /
| Issues addressed (#s or URLs) | /
| BC breaks? | N
| Deprecations? | N

#### Description:
This PR removes the video button from editors which compose email content, because video cannot be sent via email and because the button doesn't work there.

#### Steps to test this PR:
1. Apply this PR.
2. Compile prod assets or use the dev mode.
3. Make sure that the video button is gone in
  - Email -> content tab
  - Email -> email builder
  - Contact -> send email
4. But the video button is in the page editor and builder and it works there.

#### Steps to reproduce the bug:
1. You can try to use the video button in the email editor. It will throw a JS error.
